code/script.js

```diff
@@ -218,108 +227,176 @@ const generateBoard = () => {
           <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
         </div>
       </div>
-    `
-  })
-}
+    `;
+  });
+};
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  secret =
+    charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
+};
+
+// Function to add to counter after question has been asked
+const add = () => {
```

Comment:
Maybe you could have chosen a different name than add -- it isn´t clear for me at the first sight what it stands for.